code-challenges.test.js

```diff
@@ -26,37 +26,171 @@ const numbersArray2 = [24, 27, 30, 33, 36]
+    } else {
+      return `${obj.number} is not divisible by three`;
+    }
+  };
+  
+
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // Pseudo code:
+// input: an array of words
+// output: a single array with all words from both original arrays capitalized
+// process: Create a function that takes an array as a parameter.  Set the function up to concat both given arrays into one single array. Will need to create another constand to facilitate the array merger
+//create another const that will iterate over the entire array using the .map method beginning at the 0 index using charAt, then by using word.toUpperCase to go through the entire array by not indicating a starting index and using .splice to return at the beginning of the array and slice to start at the beginning array and leaving the parenthisis blank to return the last index of the array
 
 // a) Create a test with expect statements for each of the variables provided.
 
+
+describe("capitalizeWords", () => {
+  it("should capitalize all words in the array", () => {
+    const result = capitalizeWords(randomNouns1.concat(randomNouns2));
+    expect(result).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database", "Chopsticks", "Mango"]);
+  });
+});
+
+
+// const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+
+// Good failure
+// FAIL  ./code-challenges.test.js
+// evenThree
+//   ✓ should return a string indicating if the number is divisible by three or not (2 ms)
+// capitalizeArr1
+//   ✕ takes in an array of strings named randomNoms and returns and array with all values capitolized
+// indexVowels
+//   ✓ takes in a string and logs the index of the first vowel
+
+// ● capitalizeArr1 › takes in an array of strings named randomNoms and returns and array with all values capitolized
+
+//   ReferenceError: capitalizeWords is not defined
+
+
+
+
+
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
 // b) Create the function that makes the test pass.
 
+// Write the code and see the test pass
+
+// PASS  ./code-challenges.test.js
+// evenThree
+//   ✓ should return a string indicating if the number is divisible by three or not (2 ms)
+// capitalizeWords
+//   ✓ should capitalize all words in the array
+// indexVowels
+//   ✓ takes in a string and logs the index of the first vowel (1 ms)
+const capitalizeWords = (arr) => {
+  const joinedArr = randomNouns1.concat(randomNouns2);
+  const result = joinedArr.map(word => word.charAt(0).toUpperCase() + word.slice(1));
+  return result;
+}
```

Comment:
Your pseudo code does not match the code you used for your solution. Make sure you are explaining each step